html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationWrixLocations.js

```diff
@@ -0,0 +1,1019 @@
+                      { value: 'Indian/Maldives', text: 'Maldives' },
+                      { value: 'Indian/Mauritius', text: 'Mauritius' },
+                      { value: 'Indian/Reunion', text: 'Reunion' }
+                    ]
+                  },
+                  {
+                    group: 'Pacific',
+                    items: [
+                      { value: 'Pacific/Apia', text: 'Apia' },
+                      { value: 'Pacific/Auckland', text: 'Auckland' },
+                      { value: 'Pacific/Bougainville', text: 'Bougainville' },
+                      { value: 'Pacific/Chatham', text: 'Chatham' },
+                      { value: 'Pacific/Chuuk', text: 'Chuuk' },
+                      { value: 'Pacific/Easter', text: 'Easter' },
+                      { value: 'Pacific/Efate', text: 'Efate' },
+                      { value: 'Pacific/Enderbury', text: 'Enderbury' },
+                      { value: 'Pacific/Fakaofo', text: 'Fakaofo' },
+                      { value: 'Pacific/Fiji', text: 'Fiji' },
+                      { value: 'Pacific/Funafuti', text: 'Funafuti' },
+                      { value: 'Pacific/Galapagos', text: 'Galapagos' },
+                      { value: 'Pacific/Gambier', text: 'Gambier' },
+                      { value: 'Pacific/Guadalcanal', text: 'Guadalcanal' },
+                      { value: 'Pacific/Guam', text: 'Guam' },
+                      { value: 'Pacific/Honolulu', text: 'Honolulu' },
+                      { value: 'Pacific/Kiritimati', text: 'Kiritimati' },
+                      { value: 'Pacific/Kosrae', text: 'Kosrae' },
+                      { value: 'Pacific/Kwajalein', text: 'Kwajalein' },
+                      { value: 'Pacific/Majuro', text: 'Majuro' },
+                      { value: 'Pacific/Marquesas', text: 'Marquesas' },
+                      { value: 'Pacific/Nauru', text: 'Nauru' },
+                      { value: 'Pacific/Niue', text: 'Niue' },
+                      { value: 'Pacific/Norfolk', text: 'Norfolk' },
+                      { value: 'Pacific/Noumea', text: 'Noumea' },
+                      { value: 'Pacific/Pago_Pago', text: 'Pago_Pago' },
+                      { value: 'Pacific/Palau', text: 'Palau' },
+                      { value: 'Pacific/Pitcairn', text: 'Pitcairn' },
+                      { value: 'Pacific/Pohnpei', text: 'Pohnpei' },
+                      { value: 'Pacific/Port_Moresby', text: 'Port_Moresby' },
+                      { value: 'Pacific/Rarotonga', text: 'Rarotonga' },
+                      { value: 'Pacific/Tahiti', text: 'Tahiti' },
+                      { value: 'Pacific/Tarawa', text: 'Tarawa' },
+                      { value: 'Pacific/Tongatapu', text: 'Tongatapu' },
+                      { value: 'Pacific/Wake', text: 'Wake' },
+                      { value: 'Pacific/Wallis', text: 'Wallis' }
+                    ]
+                  }
+                ]
```

Comment:
I hope those will eventually be provided by the server .. ?

Reply:
I expect with OPTIONS